app/Actions/Install/PermissionsChecker.php

```diff
@@ -3,7 +3,6 @@
 namespace App\Actions\Install;
 
 use function Safe\preg_match;
-use function Safe\substr;
```

Comment:
I have seen this being removed a couple of times now. So I am finally going to ask: why? What has changed that his isn't required anymore?

Reply:
With Laravel 9, I am also upgrading to nunomaduro/larastan 2 and thecodingmachine/safe 2.
As a result the rules of phpstan are also updated. phpstan now indicates that Safe\substr is no longer need if you are using php8. Before we were using thecodingmachine/safe 1 which was designed for php 7.2+.